packages/driver/cypress/e2e/commands/assertions.cy.js

```diff
@@ -639,11 +636,11 @@ describe('src/cy/commands/assertions', () => {
         cy.get('button:first').should('have.class', 'does-not-have-class')
       })
 
-      it('has a pending state while retrying queries', (done) => {
+      it('has a pending state while retrying queries', function (done) {
         cy.on('command:retry', (command) => {
-          const [getLog, shouldLog] = cy.state('current').get('logs')
+          const [getLog, shouldLog] = this.logs
 
-          expect(getLog.get('state')).to.eq('pending')
+          expect(getLog.get('state')).to.eq('passed')
```

Comment:
When an assertion fails and the command is requried, do we set this get status back to pending? Or how are we showing this subject changed from the yielded value? Console props?